### PR TITLE
Updated vital signs and streamlined oncocore a bit. Upped the mCODE version to 0.8

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

**Comment by markkramerus** [comment]
Only ES6 export errors, as expected. Should be good to go.
444 errors (shr-es6-export)
2055 warnings (shr-text-input, shr-json-export, shr-fhir-export, shr-json-schema-export)

File: spec/cp-mcode.txt [hunk 1]
```diff
@@ -0,0 +1,196 @@
+Grammar:        ContentProfile 1.0
```
Comment (kjmahalingam) [inline spec/cp-mcode.txt]: This file is great, but please rename it to `mcode_cp.txt` to fall in line with convention. The importer will read in content profiles based on that `_cp.txt` suffix, similarly to how it reads in value set and mapping files.

File: spec/ig-mcode-config.json [hunk 2]
```diff
@@ -1,26 +1,43 @@
 {
-    "projectName": "HL7 FHIR Implementation Guide: minimal Common Oncology Data Elements (mCODE), Release 0.7 - US Realm (Draft for Comment 1)",
+    "projectName": "HL7 FHIR Implementation Guide: minimal Common Oncology Data Elements (mCODE), Release 0.8 - US Realm (Draft for Comment 1)",
     "projectShorthand": "oncocore",
     "projectURL": "http://hl7.org/fhir/us/projectURL/",
     "fhirURL": "http://hl7.org/fhir/us/shr/",
     "entryTypeURL": "http://hl7.org/fhir/us/entryTypeURL/spec/",
     "fhirTarget": "FHIR_DSTU_2",
+    "contentProfile": "cp-mcode",
```
Comment (kjmahalingam) [inline spec/ig-mcode-config.json]: Falling in line with my previous comment, please change this line to `"contentProfile": "mcode_cp.txt"` and this should be as expected by the content profile importer.

File: spec/cp-mcode.txt (hunk 1 above)
Comment (markkramerus) [reply spec/cp-mcode.txt]: Ugh. Terrible convention to rely on file name to determine the type.

Comment (markkramerus) [reply spec/cp-mcode.txt]: Everything besides the extension is user territory.

Comment (kjmahalingam) [reply spec/cp-mcode.txt]: I agree with that assessment, and actually, I think I was wrong. I'm pretty sure that the _vs and _map and _cp file endings are only necessary when using the ANTLR test parser, and not our actual tool chain. I just renamed the file back to what you have, and it worked, so my comment was invalid.

That all said, @markkramerus, I **do** still think that renaming the file to `mcode_cp.txt` is probably still best, just to follow the stylistic convention of the rest of the spec files.

File: spec/ig-mcode-config.json (hunk 2 above)
Comment (kjmahalingam) [reply spec/ig-mcode-config.json]: If you do end up choosing not to rename the file, which I'll leave up to you, you do at least have to add the `.txt` extension to this field. For example, `"contentProfile: "cp-mcode.txt"`.

File: spec/cp-mcode.txt (hunk 1 above)
Comment (markkramerus) [reply spec/cp-mcode.txt]: The reason for starting with cp- is the same as starting filenames with ig- or shr- or oncocore- . When viewed alphabetically, the files are grouped by the way they make sense to the user.

File: spec/shr_lab_panels_vs.txt [hunk 3]
```diff
@@ -0,0 +1,106 @@
+Grammar:	ValueSet 5.0
+Namespace:	shr.lab
+
+CodeSystem:     LNC     = http://loinc.org
+
+ValueSet:       CBCMustSupportPanelMembersVS
```
Comment (mlterryMitre) [inline spec/shr_lab_panels_vs.txt]: Move `LNC#751-8  	"Neutrophils [#/​volume] in Blood by Automated count"` from Optional to MustSupport list.

File: spec/shr_lab_panels_vs.txt [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+LNC#35332-6  	"Band form neutrophils/​100 leukocytes in Blood by Automated count"
+LNC#19023-1  	"Granulocytes/​100 leukocytes in Blood by Automated count"
+LNC#42250-1  	"Variant lymphocytes/​100 leukocytes in Blood by Automated count"
+LNC#58409-4  	"Other cells/​100 leukocytes in Blood by Automated count"
+LNC#30229-9  	"Band form neutrophils [#/​volume] in Blood by Automated count"
+LNC#43743-4  	"Variant lymphocytes [#/​volume] in Blood by Automated count"
+LNC#51383-8  	"Leukocytes other [#/​volume] in Blood by Automated count"
+LNC#58443-3  	"Other cells [#/​volume] in Blood by Automated count"
+
+ValueSet:       CMPMustSupportPanelMembersVS
```
Comment (mlterryMitre) [inline spec/shr_lab_panels_vs.txt]: Move the following from the optional list to must-support list.
LNC#33914-3  	"Glomerular filtration rate/​1.73 sq M.predicted [Volume Rate/​Area] in Serum or Plasma by Creatinine-based formula (MDRD)"
LNC#50044-7  	"Glomerular filtration rate/​1.73 sq M predicted among females [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#48642-3  	"Glomerular filtration rate/​1.73 sq M predicted among non-blacks [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#48643-1  	"Glomerular filtration rate/​1.73 sq M predicted among blacks [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#1968-7  	"Bilirubin.direct [Mass/​volume] in Serum or Plasma"

File: spec/ig-mcode-config.json (hunk 2 above)
Comment (markkramerus) [reply spec/ig-mcode-config.json]: Thanks, I added .txt and kept cp-mcode

File: spec/shr_lab_panels_vs.txt (hunk 3 above)
Comment (markkramerus) [reply spec/shr_lab_panels_vs.txt]: Did you realize that this puts two neutrophil tests on the MS list?
```
LNC#770-8  	    "Neutrophils/​100 leukocytes in Blood by Automated count"
LNC#751-8  	    "Neutrophils [#/​volume] in Blood by Automated count"
```

File: spec/shr_lab_panels_vs.txt (hunk 4 above)
Comment (markkramerus) [reply spec/shr_lab_panels_vs.txt]: done

**Comment by mlterryMitre** [comment]
looks good. merging.

File: spec/cp-mcode.txt (hunk 1 above)
Comment (kjmahalingam) [reply spec/cp-mcode.txt]: Okay -- fine with me.

**Pull request merged**